Packs/Intezer/Integrations/IntezerV2/IntezerV2.py

```diff
@@ -1,6 +1,9 @@
 from collections import defaultdict
 from http import HTTPStatus
 from typing import Callable
+from typing import Dict
+from typing import List
+from typing import Tuple
```

Comment:
```suggestion
from typing import Callable, Dict, List, Tuple
```

Reply:
Hi, I prefer to have it multiline. It can reduce the chances of git conflicts in case two people are working on the same file when adding new imports. WDYT?